app/helpers/application_helper.rb

```diff
@@ -176,7 +176,8 @@ def tag_link(tag)
     link_to tag.tag,
       tag_path(tag),
       class: [tag.css_class, filtered_tags.include?(tag) ? "filtered" : nil],
-      title: tag.description
+      title: tag.description,
+      id: "tag-#{tag.tag}"
```

Comment:
This helper is used throughout the site, so adding an anchor here is going to put duplicate anchors to a lot of pages, like any list of stories.

Reply:
Yeah, I noticed that and I kind of expected this comment.

Should I allow the `tag_link` to accept an options object then? Cause for the "Filter tags" page, I need to pass this function an `aria-label` to correctly label filtered tags.